src/components/AllTags/TagsSelected.vue

```diff
@@ -83,60 +70,63 @@ export default {
       },
     },
   },
-  methods:{
+  methods: {
     queryParamTags(sectionName) {
       switch (sectionName) {
       case "taxonomic range":
-        return "taxonomies"
+        return "taxonomies";
       case "user_defined_tag":
-        return "userDefinedTags"
+        return "userDefinedTags";
       default:
-        return sectionName
+        return sectionName;
       }
     },
-    removeTag(tagParams){
-      let id = tagParams[0]
-      let label = tagParams[1]
-      let sectionName = tagParams[2]
-      this.selectedTags = this.selectedTags.filter(el => el.id !== id);
-      Object.keys(this.getQueryParams).forEach(key => {
-        if(key === this.queryParamTags(sectionName)) {
-          this.selectedQueryParam = this.getQueryParams[this.queryParamTags(sectionName)].filter(el => el !== label)
-          this.getQueryParams[key] = this.selectedQueryParam
+    /* v8 ignore start */
```

Comment:
Is there any particular reason this part is ignored now?

Reply:
The test for this method has been added